client/src/curriculum/module.scss

```diff
@@ -0,0 +1,58 @@
+@use "../ui/vars" as *;
+
+.curriculum-content-container.with-sidebar,
+.curriculum-content-container {
```

Comment:
This seems like it is redundant, or do some of the classes below get overridden for classes other than `.with-sidebar`?